powershell/admin-center/Microsoft.PowerApps.Administration.PowerShell.Samples.psm1

```diff
@@ -1400,6 +1400,218 @@ function UpdatePolicyEnvironmentsForTeams
+    {
+        Write-Host "The specified environment is not managed."
+        return
+    }
+
+    $governanceConfiguration.protectionLevel = "Basic"
+    
+    $response = Set-AdminPowerAppEnvironmentGovernanceConfiguration -EnvironmentName $EnvironmentId -UpdatedGovernanceConfiguration $GovernanceConfiguration
+    if ($response.Code -ne 202)
+    {
+        Write-Host "Failed to disable Management for the specified environment."
+        Write-Host $response.Internal.Message
+        return
+    }
+    
+    Write-Host "Disabled Management for the specified environment."    
+}
+
+function IncludeInsightsForManagedEnvironmentInWeeklyEmailDigest
+{
+    <#
+     .SYNOPSIS
+     Includes insights for the specified managed environment from weekly email digest.
+     .DESCRIPTION
+     The IncludeInsightsForManagedEnvironmentInWeeklyEmailDigest cmdlet includes insights for the specified managed environment from weekly email digest.
+     Use Get-Help IncludeInsightsForManagedEnvironmentInWeeklyEmailDigest -Examples for more details.
+     .PARAMETER EnvironmentId
+     The id (usually a GUID) of the environment.
+     .EXAMPLE
+     IncludeInsightsForManagedEnvironmentInWeeklyEmailDigest -EnvironmentId 8d996ece-8558-4c4e-b459-a51b3beafdb4
+     Includes insights for managed environment with id 8d996ece-8558-4c4e-b459-a51b3beafdb4 in the weekly email digest.
+    #>
+    param
+    (
+        [Parameter(Mandatory = $true)]
+        [string]$EnvironmentId
+    )
+
+    Write-Host "Retrieving environment."
+
+    $environment = Get-AdminPowerAppEnvironment -EnvironmentName $EnvironmentId
+    if ($environment -eq $null)
+    {
+        Write-Host "No environment was found with the given id."
+        return
+    }
+
+    $governanceConfiguration = $environment.Internal.properties.governanceConfiguration
+    $governanceConfiguration = CoalesceGovernanceConfiguration -GovernanceConfiguration $governanceConfiguration
+    if ($governanceConfiguration.protectionLevel -ne "Standard")
+    {
+        Write-Host "The specified environment is not managed."
+        return
+    }
+
+    if ($governanceConfiguration.settings.extendedSettings.excludeEnvironmentFromAnalysis -ne "True")
+    {
+        Write-Host "The specified environment is already included in weekly email digest."
+        return
+    }
+    
+    $governanceConfiguration.settings.extendedSettings.excludeEnvironmentFromAnalysis = "false"
+
+    $response = Set-AdminPowerAppEnvironmentGovernanceConfiguration -EnvironmentName $EnvironmentId -UpdatedGovernanceConfiguration $GovernanceConfiguration
+    if ($response.Code -ne 202)
+    {
+        Write-Host "Failed to include insights for the specified environment in weekly email digest."
+        Write-Host $response.Internal.Message
+        return
+    }
+    
+    Write-Host "Included insights for the specified environment in weekly email digest."    
+}
+
+function ExcludeInsightsForManagedEnvironmentInWeeklyEmailDigest
```

Comment:
ExcludeInsightsForManagedEnvironmentsInWeeklyEmailDigest (plural for Environments)